tests/unit/CMakeLists.txt

```diff
@@ -22,6 +22,10 @@ set(subdirs
     util
    )
 
+if(HPX_WITH_NETWORKING)
+  set(subdirs ${subdirs} parcelset)
+endif()
```

Comment:
Please remove `parcelset` from the list above.

Reply:
Ping @biddisco?

Reply:
doh! fixed.